scss/atoms/typography/_typography.scss

```diff
@@ -21,40 +21,46 @@ body {
 }
 
 h1 {
-	font-size: em( 67px ); // 54px works nicely, as well
-	font-weight: 700;
-	line-height: 1;
+	@include heading($h1);
 }
 
 h2 {
-	font-size: em( 36px );
-	font-weight: 600;
-	line-height: 1;
+	@include heading($h2);
 }
 
 h3 {
-	font-size: em( 30px );
-	font-weight: 600;
-	line-height: 1;
+	@include heading($h3);
 }
 
 h4 {
-	font-size: em( 24px );
-	font-weight: 600;
-	line-height: 1;
+	@include heading($h4);
 }
 
 h5 {
-	font-size: em( 20px );
-	font-weight: 600;
-	line-height: 1;
+	@include heading($h5);
 }
 
 h6 {
-	font-size: em( 16px );
-	font-weight: 600;
-	line-height: 1;
-	text-transform: uppercase;
+	@include heading($h6);
+}
+
+p {
+	text-align: map-get($p,textAlignment);
+	margin: em(map-get($p,margin)) 0;
```

Comment:
have both values for margin be in the variable, not just the top/bottom.  Otherwise people have no way to control the left/right margin.